torch/_subclasses/meta_utils.py

```diff
@@ -127,18 +127,24 @@ def del_ten():
 
     # NB: doesn't actually return a storage, because meta storage is
     # not supported
-    def meta_storage(self, s):
+    def meta_storage(self, s, callback):
         # NB: TypedStorage is freshly allocated and cannot be used as hash
         # key index.
 
         # Use a Weak Ref to s in order to not leak memory
         swr = StorageWeakRef(s)
         if swr not in self.storage_memo:
-            self.storage_memo[swr] = torch.empty(s.size(), dtype=s.dtype, device="meta")
+            self.storage_memo[swr] = (
+                callback(
+                    lambda: torch.empty(s.size(), dtype=torch.uint8, device="meta")
```

Comment:
`s.dtype ->torch.uint8` - don't we need `s.size() * s.element_size()` if we're going to use `uint8`

Reply:
No because it's untyped storage so already reporting size in bytes. Maybe could use nbytes() to ne clearer.